### ISSUE TITLE
Importing in fava and the auto_accounts plugin

### ISSUE THREAD

**Issue by Ramblurr** [beancount/fava#1188]
First, I swear this worked in the past, but now I am having issues.

When I use the fava import feature, I thought I used to be able to create accounts on the fly (because I have the beancount auto_accounts plugin enabled). But now I am getting the error "Should be one of the declared accounts".

Am I wrong and this never worked? Or did something change in the past year or so?

**Comment by yagebu** [comment]
> Am I wrong and this never worked? Or did something change in the past year or so?

Yep this used to work since the account names weren't checked against the list of existing accounts, but they are now. This doesn't really play well together with auto_accounts...
